MNIST/MNIST.swift

```diff
@@ -115,7 +115,7 @@ func train(_ parameters: inout MNISTParameters, epochCount: Int32) {
             totalGuesses += Int(minibatchSize)
         }
         print("""
-              Accuracy: \(correctGuesses)/\(totalGuesses) \
+              Epoch[\(epoch)] Accuracy: \(correctGuesses)/\(totalGuesses) \
```

Comment:
How about this format?
```suggestion
              [Epoch \(epoch)] Accuracy: \(correctGuesses)/\(totalGuesses) \
```

Reply:
done